classes/class4/MyHomework/app.js

```diff
@@ -0,0 +1,48 @@
+var express = require('express');
+var path = require('path');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var exphbs  = require('express-handlebars');
+var mongoose = require('mongoose');
+var PORT = process.env.PORT || 3000;
+var mongoURI = process.env.MONGOURI || "mongodb://localhost/test";
+mongoose.connect(mongoURI);
+
+var app = express(); 
+var populateDatabase = require('./models/populateDatabase');
+var routes = require('./routes/burgers');
+populateDatabase.main();
+
+app.engine("handlebars", exphbs({defaultLayout: "main"}));
+app.set("view engine", "handlebars");
+app.use(logger("dev"));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, "public")));
+
+app.get('/', function(req,res) {
+	res.send("Welcome to Jessica's Burgers");
```

Comment:
I know it's tempting to put short routes in app.js but it's better if they're all together in a route file, and not in app.js.